src/main.js

```diff
@@ -174,10 +174,17 @@ function createWindow () {
 
   var menuBar = Menu.buildFromTemplate([
     {
-      label: 'File',
+      label: '&File',
       submenu: [
+        { role: 'undo' },
+        { role: 'redo' },
+        { type: 'separator' },
+        { role: 'cut' },
+        { role: 'copy' },
+        { role: 'paste' },
+        { type: 'separator' },
```

Comment:
Do I understand correctly that the typical roles like this are automatically given keyboardable labels and don't require separate definitions akin to `label: '&Undo'`?

Reply:
I don't think so. They would likely need the special label.